lambda/lambda_function.py

```diff
@@ -4,17 +4,17 @@
 # Please visit https://alexa.design/cookbook for additional examples on implementing slots, dialog management,
 # session persistence, api calls, and more.
 
-import logging
-import ask_sdk_core.utils as ask_utils
 import json
+import logging
 
-from ask_sdk_core.skill_builder import SkillBuilder
-from ask_sdk_core.dispatch_components import AbstractRequestHandler
-from ask_sdk_core.dispatch_components import AbstractExceptionHandler
+import ask_sdk_core.utils as ask_utils
```

Comment:
if we are not importing python system modules or entire pip libraries, I would prefer we always use the from `from x import y as z`

Reply:
Fixed :+1: was like that initially so it didnt change :)